scripts/src/test.sh

```diff
@@ -30,7 +30,7 @@ EOF
 }
 
 workspace_test() {
-  cargo nextest run --workspace "$@" --profile ci
+  cargo nextest run --workspace "$@" --profile ci --exclude gear-program
```

Comment:
excluded tests for `gear-program` for a while -- it should use node from actual repo